src/test/kotlin/abilities/CommunicateViaDidcomm.kt

```diff
@@ -69,6 +71,12 @@ open class CommunicateViaDidcomm(val mediatorPeerDid: String, serviceEndpoint: S
         return didcommMessage
     }
 
+    fun unpackLastSignedDidcommMessagePayload(): Message {
```

Comment:
This can be improved after we clarify this
https://github.com/decentralized-identity/didcomm-messaging/issues/446

(just in case if you want to add a note)